File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L4/05/inputoutput.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let rad: number;
let panjang: number;

function input(): void {
  rl.question('Enter the radius and length of a cylinder: ', (input) => {
    const [r, l] = input.split(' ').map(Number);
    rad = r;
    panjang = l;
    rl.close();
  });
}

function hitArea(): number {
  return rad * rad * 3.14159;
}

function main(): void {
  input();

  const a = hitArea();
  const vol = a * panjang;

  console.log(`The area is ${a}`);
  console.log(`The volume of the cylinder is ${vol}`);
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L1/05/inputoutput.kt
package IR

import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`)

    // masukan silinder
    print("Enter the radius and length of a cylinder: ")
    val radius = input.nextDouble()
    val length = input.nextDouble()

    // perhitungan
    val area = radius * radius * 3.14159
    val volume = area * length

    // cetak hasil
    println("The area is $area")
    println("The volume of the cylinder is $volume")
}

